catalog/core/catalog-core-camelcomponent/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -17,11 +17,11 @@
   http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd"
         >
 
-    <reference id="transformerMapper" interface="ddf.mime.MimeTypeToTransformerMapper"/>
+    <reference id="transformerMapper" interface="ddf.mime.MimeTypeToTransformerMapper" availability="optional"/>
```

Comment:
@vinamartin, I think the only one in this file that needs `optional` added is the CatalogFramework.